content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a certificate in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory. The following example downloads and installs the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin assuming the environment variables are set for the plugin version and install path for your development computer.
+
+{{% alert title="Warning" color="primary" %}}
+The following example does not work unless you set the environment variables, and other plugins may follow a different process for downloading. You also need to know the specific location `{NOTATION_LIBEXEC}/plugins/` on your development computer. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/concepts/directory-structure" >}}).
```

Comment:
Just curious about the `{NOTATION_LIBEXEC}` formatting we use in those examples. Is this standard? I am more used to the `$NOTATION_LIBEXEC` or `${NOTATION_LIBEXEC}` as env variables and using `<plugin-name>` for things that need to be replaced in the commands. Just wanted to confirm that this is what our readers will expect.

Reply:
i kept the convention from https://notaryproject.dev/docs/concepts/directory-structure/#structure, but i can refactor if you think `<name>` is clearer

Reply:
lets leave as is for now and can refactor later if needed